### DISCUSSION TITLE
Face global flags issue while install

### DISCUSSION THREAD

**Discussion by chienggc** [django-wiki/django-wiki#1234] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/103575743/207843216-d04ba413-4d40-4162-8888-f4d3abadcd9a.png)
hitting error "global flags not at the start of the expression at position 7" when navigate to my locahost http://127.0.0.1:8000/
can someone help to solve this issue?

**Comment by benjaoming** [comment]
django-wiki doesn't work with Python 3.11 yet, see: https://github.com/django-wiki/django-wiki/issues/1232

**Comment by oscarmcm** [comment, marked as answer]
Fixed on #1243
